ethcore/src/executive.rs

```diff
@@ -184,18 +185,20 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 			schedule: schedule,
 			depth: 0,
 			static_flag: false,
+			transaction_checkpoint_index: None,
```

Comment:
This needs a bit of comment I think. From what I understand it is to manage reentrant call by addressing the first checkpoint initial values; if it is the case renaming to `first_transaction_checkpoint_index` also makes sense.